<!--
  Works with rocon_gateway_hub/hub.launch. 
  
  This one has no auto-discovery mode. Instead it uses manual direct
  configuration with the typical default settings for a local ubuntu 
  redis server.
-->
<launch>
  <node pkg="rocon_gateway" type="gateway.py" name="gateway" output="screen">
    <rosparam command="load" file="$(find rocon_gateway)/param/default.yaml" />
    <rosparam param="hub_uri">http://localhost:6379</rosparam>
  </node>
</launch>
